example/functions-example/HttpTriggerToddysmExample/function.json

```diff
@@ -0,0 +1,20 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
```

Comment:
adding "delete" would be helpful

Reply:
would we add patch? Just a question in regards to last quarter for the different responses.

Reply:
@nataliagourova and @theKunte depends on what endpoint this function implements. If it is only the `/users` you need to think whether `PUT/PATCH` and `DELETE` are appropriate.